compiler/aarch64/codegen/OMRCodeGenerator.cpp

```diff
@@ -72,7 +72,9 @@ OMR::ARM64::CodeGenerator::initialize()
    _linkageProperties = &cg->getLinkage()->getProperties();
 
    cg->setStackPointerRegister(cg->machine()->getRealRegister(_linkageProperties->getStackPointerRegister()));
+#ifdef J9_PROJECT_SPECIFIC
```

Comment:
Please don't add `#ifdef J9_PROJECT_SPECIFIC` to AArch64 code.
We tried to get rid of it in OMR when we started the AArch64 JIT work.

Reply:
Fair enough, I dropped last two commits. Thanks!